src/rapids_dependency_file_generator/schema.json

```diff
@@ -0,0 +1,160 @@
+ {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/rapidsai/dependency-file-generator/v1.0.0/src/rapids_dependency_file_generator/schema.json",
```

Comment:
Could we automatically update this URL to match the release version?

Reply:
@ajschmidt8 you have more familiarity with what we can do with `semantic-release`, do you have any recommendations on how we could handle this? Specifically, what we want is for the version number in the `"$id"` URL to be updated to match the current version during a release. My concern was that the PR to update the version number would itself trigger a new version, thus immediately rendering the version in the URL out of date.

Reply:
@vyasr, @csadorf, yes. we can do that with https://github.com/semantic-release/git.

We'll have to figure out how to configure it, but it will commit "release assets" back to the repository (with a `[skip ci]` tag) after a release has occurred.

In this case, `schema.json` would be categorized as a "release asset" since it depends on the release version.